.github/ISSUE_TEMPLATE/crate-build-failure.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Crate Build Failure
+description: My crate failed to build in docs.rs.
+labels: build-failure
+body:
+  - type: markdown
+    attributes:
+      value: |
+        If you need a system dependency added for your crate to build,
```

Comment:
Now that we have access to markdown we could have this in the same line and use proper links instead of having the full URLs.